packages/shared/src/components/auth/SocialRegistrationForm.tsx

```diff
@@ -58,6 +66,8 @@ export const SocialRegistrationForm = ({
   const { user } = useContext(AuthContext);
   const [nameHint, setNameHint] = useState<string>(null);
   const [usernameHint, setUsernameHint] = useState<string>(null);
+  const [twitterHint, setTwitterHint] = useState<string>(null);
+  const isAuthorOnboarding = trigger === AuthTriggers.Author || true;
```

Comment:
This will always be true.

Probably this was for dev mode.

Reply:
Oh yes, indeed! sharp eyes 🔭

Reply:
Happens to me always, so I know 🤣